chains/atleta-sepolia-testnet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "atleta-sepolia-testnet",
```

Comment:
From docs, I see that this chain is L1. So, should `alias` include `-sepolia`?

Reply:
Removed -sepolia from alias